main.pm

```diff
@@ -99,6 +99,10 @@ if (get_var("LIVETEST")) {
     $testapi::username = "root";
     $testapi::password = '';
 }
+elsif (get_var("AUTOYAST") || get_var("SUPPORT_SERVER") ) {
+    $testapi::username = "root";
+    $testapi::password = "root";
+}
```

Comment:
There are autoyast tests besides the ones you're writing. The autoyast tests being put together by QA use bernhard/nots3cr3t like our usual tests, so I really dont like this

Users shouldn't use root unless they need it, so our tests shouldn't use root unless they need it